tests/upgrades/test_yum_plugins.py

```diff
@@ -47,8 +45,8 @@ class Scenario_yum_plugins_count(APITestCase):
     Test Steps:
 
         1. Before Satellite upgrade.
-        2. Create Organization, Location and LifecycleEnvironment.
-        3. Upload Manifest, enable/sync 'base os RHEL7' and tools repos.
+        2. Create LifecycleEnvironment.
+        3. Get details of 'base os RHEL7' and tools repos.
```

Comment:
You need not to modify this as the goal of docstrings is not to tell what we are testing it but how the test scenario should be in general.

Reply:
@jyejare okay .. just to clear ...do you want to keep old doc string as it is ?

Reply:
Yes

Reply:
@jyejare but that is not true anymore i.e. now we are not `Create Organization, Location ` and `Upload Manifest, enable/sync repos` here

Reply:
> @jyejare but that is not true anymore 

This is not anymore true, but in automation. Think when this docstrings will be read as test in polarion, 'Get details of 'base os RHEL7' and tools repos.' sounds weird.

> i.e. now we are not `Create Organization, Location ` and `Upload Manifest, enable/sync repos` here

Not us but any person reading docstring and has no idea of upgrade images or org1, he will end up in confusion.

Instead I will suggest u to do this:
```suggestion
        3. Enable/sync 'base os RHEL7' and tools repos.
```